dhcp/config.ml

```diff
@@ -0,0 +1,21 @@
+open Mirage
+
+let main = foreign "Unikernel.Main" (console @-> kv_ro @-> network @-> clock @-> job)
+
+let disk = crunch "files"
+
+(* TODO: workaround a bug in the command-line tool by adding Clock
+   for Unix (this is pulled in as an implicit dependency in Xen) *)
```

Comment:
is this reported somewhere, that `mirage-clock-unix` needs to be manually pulled in? does it happen with mirage-master or the latest release? @Drup @samoht